molgenis-data/src/main/java/org/molgenis/data/support/AbstractRepositoryCollection.java

```diff
@@ -37,6 +37,12 @@ public void deleteRepository(EntityType entityType)
 	}
 
 	@Override
+	public void updateRepository(EntityType entityType, EntityType updatedEntityType)
+	{
+		throw new UnsupportedOperationException();
```

Comment:
Is a bit weird construction, I'd expect an abstract method or a do-nothing implementation. But I see you're simply doing what we're already doing :)